.github/workflows/test-push-live-k8s-cluster.yml

```diff
@@ -16,33 +16,25 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Set up Helm
-        uses: azure/setup-helm@v1
+      - name: Setup Python
+        uses: actions/setup-python@v2.2.2
         with:
-          version: v3.4.0
+          python-version: 3.7
+          architecture: x64
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8 pytest
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
 
       - uses: actions/setup-python@v2.2.2
         with:
           python-version: 3.7
           architecture: x64
```

Comment:
Why do we have to run `setup-python` twice?

Reply:
Removed the second setup-python